code.js

```diff
@@ -1,9 +1,24 @@
 function factorial(n) {
-    if(n === 0) return 1;
+    if (n === 0) return 1;
     else return n * factorial(n - 1);
 }
 
 function e(n) {
-    if(n === 0) return 1;
+    if (n === 0) return 1;
     else return 1.0 / factorial(n) + e(n - 1);
 }
+
+function bottomUp(n) {
```

Comment:
Can you change the existing function please so that the code is tested (or add this function to the tests)?

Reply:
Ok I think I added the test for bottomUp correctly. I added a screenshot in the readme for how I got the value of bottomUp(10)